applications/README.md

```diff
@@ -0,0 +1,25 @@
+# Applications
+This directory contains CDK-based serverless applications.
+
+## Prerequisites
+
+* Either install esbuild (`npm install -g esbuild`); otherwise the build process will default to using Docker images which requires Docker to be installed and running locally. Note that esbuild is _much_ faster than Docker
+* You may need to run `cdk bootstrap` before provisioning infrastructure
+
```

Comment:
Do you expect all lambda's to be written in TS James, or can they be written in anything? If so, it would be good to document why you've chosen TS over plain'ol JS or any of the other supported languages in here. Can of worms is now open 🪱

Reply:
I think we would favour TS as the 'standard best practice' language for Lambdas, however they could be written in anything. They can still be provisioned using CDK regardless of what language they are written in, which is pretty neat. You're right though we should probably document the thought process behind it, potentially in a separate RFC